fem/pfespace.hpp

```diff
@@ -264,6 +264,12 @@ class ParFiniteElementSpace : public FiniteElementSpace
        including the dofs for the edges and the vertices of the face. */
    virtual void GetFaceDofs(int i, Array<int> &dofs) const;
 
+   /// Return an Operator that converts L-vectors to E-vectors on each face.
+   virtual const Operator *GetFaceRestriction(ElementDofOrdering e_ordering,
+                                              FaceType type,
+                                              L2FaceValues mul = L2FaceValues::DoubleValued
+                                             ) const;
```

Comment:
I assume this parallel version is different from the serial one because of the presence of shared faces. Can you add a sentence of two in the doxygen documentation about. For example, are shared faces treated as interior faces? If so, does the returned operator handle the communication needed to get the shared face values from neighbor processors?

Reply:
I used this comment to create documentation as you suggested.